.circleci/config.yml

```diff
@@ -0,0 +1,219 @@
+defaults: &defaults
+  working_directory: ~/repo
+  environment:
+    MAVEN_OPTS: -server -showversion -XX:+PrintCommandLineFlags -XX:+UseCodeCacheFlushing -Xms1024M -Xmx2048M -XX:+CMSClassUnloadingEnabled -XX:-OmitStackTraceInFastThrow -XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:+CMSConcurrentMTEnabled -XX:+CMSParallelRemarkEnabled -XX:+UseCMSInitiatingOccupancyOnly -XX:CMSInitiatingOccupancyFraction=70 -XX:+ScavengeBeforeFullGC -XX:+CMSScavengeBeforeRemark -XX:NewSize=600m -XX:MaxNewSize=900m -XX:SurvivorRatio=10 -XX:+DisableExplicitGC -Djava.security.egd=file:/dev/./urandom
+
+version: 2
+jobs:
+  build:
+    <<: *defaults
+    docker:
+      - image: killbill/kbbuild:0.1.0
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-dependencies-{{ .Branch }}-{{ checksum "pom.xml" }}
+      - run:
+          name: Setup dependencies
+          command: |
+            if [ "${CIRCLE_BRANCH}" != "master" ]; then
+              for i in killbill-oss-parent killbill-api killbill-plugin-api killbill-commons killbill-platform; do
```

Comment:
I think you might need the `killbill-client-java` as well?

Reply:
Ah yes, good point, I'll add it.